### DISCUSSION TITLE
Easy-Move-Resize rework for Pro

### DISCUSSION THREAD

**Discussion by LiamKearn** [rxhanson/Rectangle#929] [Ideas]
While I love org.dmarcotte.Easy-Move-Resize. It's implementation is a little old and can cause some very weird UX and bugs. A modern implementation of this ubuntu style window management might be a cool feature for Rectangle pro!

I'll being grabbing a copy of Pro soon regardless thanks for the great app and quick workaround on chromium's issues with toggling AXEnhancedUserInterface causing their browser process cached version of the AXTree to be update too frequently (https://github.com/rxhanson/Rectangle/commit/075174b1753e1f06779285eae6b46203670a4fbf).

**Comment by LiamKearn** [comment]
Also any chance of a tag specifically for that? Seems big enough an issue to justify tagging even if it's a workaround.

**Comment by LiamKearn** [comment]
Just found out I asked a silly question!

This already in rectangle pro :) I can't use the mouse like I did with the other app or ubuntu but I'm happy with the new motion as well.

**Reply by rxhanson** [reply]
There are some adjustments that are on my backlog for move/resize, so when those changes become the next priority, I'll take a look at mouse button support.
